frontend-project/src/models/global.ts

```diff
@@ -1,85 +1,12 @@
-import { NoticeIconData } from '@/components/NoticeIcon';
-import { queryNotices } from '@/services/user';
-import { ConnectState } from './connect.d';
-
-export interface NoticeItem extends NoticeIconData {
```

Comment:
Wyrzuciłeś NoticeItem, czy będziemy potrzebowali w najbliższym czasie powiadomień ? 
Jeżeli tak to można by było zaadaptować te mechanizmy pod nas.

Reply:
Usuwam, bo nie będziemy. Najwyżej e-mailowe.